src/Microsoft.AspNetCore.Diagnostics/baseline.netfranework.json

```diff
@@ -1,2213 +0,0 @@
-{
```

Comment:
Did you notice that the filename of this file is wrong? What's up with that? cc @javiercn